patches/api/0178-Flip-some-Spigot-API-null-annotations.patch

```diff
@@ -83,6 +83,28 @@ index f5f41fb37fd8dbdcffe47eefa939aecdad8210c0..a09513f74b3fb9790958f40a7ebef952
      <T extends Keyed> Tag<T> getTag(@NotNull String registry, @NotNull NamespacedKey tag, @NotNull Class<T> clazz);
  
      /**
+diff --git a/src/main/java/org/bukkit/entity/LingeringPotion.java b/src/main/java/org/bukkit/entity/LingeringPotion.java
+index f124b35ec76e6cb6a1a0dc464005087043c3efd0..bafef53c1d449135f1300c8c8fbb06f482ba67e1 100644
+--- a/src/main/java/org/bukkit/entity/LingeringPotion.java
++++ b/src/main/java/org/bukkit/entity/LingeringPotion.java
+@@ -5,4 +5,5 @@ package org.bukkit.entity;
+  *
+  * @deprecated lingering status depends on only on the potion item.
+  */
++@Deprecated
```

Comment:
Wrong patch for this

Reply:
What's the right patch? I figured this one was fine because it also adjusts spigot annotations.

Reply:
I planned to rename the commit message of this patch. Will that be okay?

Reply:
Yeah, that should be fine. 

If you do edit the commit msg like you posted, make sure to apply and then rebuild patches. I don’t actually know if you can change the commit msg like that, but if it works, then it’s all good.

Reply:
I renamed the patch and changed the subject message